src/SignalR/server/Core/src/HubInvocationContext.cs

```diff
@@ -2,20 +2,41 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
+using Microsoft.AspNetCore.Authorization;
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// Context for a Hub invocation.
+    /// </summary>
     public class HubInvocationContext
     {
+        /// <summary>
+        /// Instantiates a new instance of the <see cref="HubInvocationContext"/> class.
+        /// </summary>
+        /// <param name="context">Context for the active hub connection and caller.</param>
```

Comment:
nit:Should this be "hub connection" or `HubConnection`?

Reply:
I don't think it should be `HubConnection` as that is the client side type, on the server it's a "connection to a hub" i.e. "hub connection". But I could be convinced either way